api_v3/services/CategoryEntryService.php

```diff
@@ -34,7 +34,7 @@ function addAction(KalturaCategoryEntry $categoryEntry)
 		try
 		{
 			$dbCategoryEntry = new categoryEntry();
-			$dbCategoryEntry->add($categoryEntry->entryId, $categoryEntry->categoryId);
+			$dbCategoryEntry->add($categoryEntry->entryId, $categoryEntry->categoryId, false);
```

Comment:
We want to avoid from creating new in our flow  (if already exists - just update it. if its new - create it), but there is other flow that branches with our -kLiveStream (already tested that there is no damage in this flow). 
Note: In the kLiveStream flow we need to force new creation of objects.